lib/ueberauth/strategy/facebook.ex

```diff
@@ -56,6 +53,22 @@ defmodule Ueberauth.Strategy.Facebook do
       fetch_user(conn, client)
     end
   end
+  def handle_callback!(token) when is_binary(token) do
```

Comment:
Same below, but you could that, it's just depends of what you want to do in the API. The reason I did this is because I pushed my code of ueberauth outside of a phoenix app so I didn't want to have dependencies with `Plug` but I guess it wouldn't makes sense and we could just add `Plug.Conn` everywhere and should be easier.

I don't know, I just need directions in what we should do